### PR TITLE
Add morning challenge run in Gaza

### PR DESCRIPTION
**Pull request by eliascodes**
This is really just an FYI, recording one of the morning challenges that was improvised in Gaza. It highlighted a lot of misunderstandings people had around object identity. 

This doesn't need to be merged necessarily, just wanted to create a record of it in case it's in any way useful. If not, we can close this 😃

<hr> 

EDIT: OK looks like at least two people want this in, and it's only in fragments, so I guess lets merge it if no one else objects :smile:

### CODE REVIEW

**Comment by astroash** [comment]
@eliascodes I really like this workshop and feel like there was a fair bit of misundertanding in my cohort around this. I wonder if there is space for this in future..

**Comment by shiryz** [comment]
@eliascodes it'll be great if we can get this merged, thinking of adding this as an extra next week to clarify some things.

File: coursebook/fragments/mc-objects-and-arrays/index.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+//
+// This is just an example, but you can use it to decide how you will complete
+// the functions below.
+
+
+// This function should take the buildings object (`bldngs`) and return an array
+// of objects that each describe a building. Using the `buildings` object above
+// as an example input, this function would return:
+// [
+//   {
```
Comment (shiryz) [inline coursebook/fragments/mc-objects-and-arrays/index.js]: object missing a key

File: coursebook/fragments/mc-objects-and-arrays/.index.solution.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+//
+// This is just an example, but you can use it to decide how you will complete
+// the functions below.
+
+
+// This function should take the buildings object (`bldngs`) and return an array
+// of objects that each describe a building. Using the `buildings` object above
+// as an example input, this function would return:
+// [
+//   {
```
Comment (shiryz) [inline coursebook/fragments/mc-objects-and-arrays/.index.solution.js]: id key missing

**Comment by eliascodes** [comment]
Thanks all! Any approve should feel free to merge 👍

**Pull request merged**